structure_tests/drivers/internalDriver.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2017 Google Inc. All rights reserved.
```

Comment:
I don't think it's really worth keeping an internal driver implementation. Let's just switch everything over to Docker once we get it working.

Reply:
yep agreed, just got left around from when I originally restructured the code. I'll remove it